example/Python/src/Piano/sensor_simulator.py

```diff
@@ -0,0 +1,333 @@
+from multiprocessing import Process, connection
+from threading import Thread
+from os import sys
+import os
+os.environ['PYGAME_HIDE_SUPPORT_PROMPT'] = "hide"
+try: 
+    from mingus.containers.note import Note
+    from mingus.midi import fluidsynth
```

Comment:
It seems like `fluidsynth` has to be installed separately (via a package manager perhaps). Therefore, even after installing `mingus`, I would still get `"Import Error: Failed to import 'mingus'. Try 'pip3 install mingus'"`, but I think the message should be to install `fluidsynth` (e.g., on macOS, it's `brew install fluid-synth` and most likely `sudo apt-get install -y fluidsynth` on Linux) if the `fluidsynth` import fails..

With that in mind, I'm not sure how it might work on Windows since it doesn't have a robust package manager to install `fluidsynth`.

Reply:
I added messages in the import statement according to your suggestions:

```python3
try:
    from mingus.midi import fluidsynth
except:
    if sys.platform == "darwin":
        print("Import Error: fluidsynth is missing. Try 'brew install fluidsynth'")
    elif sys.platform == "linux" or sys.platform == "linux2":
        print("Import Error: fluidsynth is missing. Try 'sudo apt-get install -y fluidsynth'")
    else:
        print("Import Error: fluidsynth is missing. ")

```

Reply:
Wow, sdl2 is pretty cool. I wonder why I didn't find that when I was looking for a way to do the Rhythm demo.  It's not clear to me why xquartz is needed if you have sdl2, since sdl2 apparently does cross-platform graphics using native libraries.

Reply:
I think installing `xquartz` might be optional: https://github.com/pygame/pygame/issues/2452#issuecomment-864364494